### ISSUE TITLE
specific item for class constructors

### ISSUE THREAD

**Issue by grahamdyson** [DevSnicket/eunice#93]
It would be clearer what the child items and dependencies are for class constructors if they had their own item.

```javascript
class Class {
	constructor() {
		called();
		function functionDeclaration() {}
	}
}
```

```yaml
- id: Class
  items:
    id: constructor
    type: method
    dependsUpon: called
    items: functionDeclaration
```

**Comment by grahamdyson** [comment]
released in [JavaScript 6.27.0](https://devsnicket.com/eunice/javascript/releases#6.27.0)

**Comment by grahamdyson** [comment]
released in [JavaScript 6.27.0](https://devsnicket.com/eunice/javascript/releases#6.27.0)
